### ISSUE TITLE
is this illumination and pose disorientation resistant?

### ISSUE THREAD

**Issue by mariya5253** [vudung45/FaceRec#41]

**Comment by rosebella18** [comment]
Yes this worked well if illumination changes and upto some extend this works for head pose too if we train well

**Comment by mariya5253** [comment]
> Yes this worked well if illumination changes and upto some extend this works for head pose too if we train well

**Comment by mariya5253** [comment]
@rosebella18  how do we train here? we are using the pretrained model already.. the already created graph and the weights.. right?
